quickstarts/microsoft.datafactory/data-factory-get-started/main.bicep

```diff
@@ -0,0 +1,199 @@
+@description('Data Factory Name')
+param dataFactoryName string = 'datafactory${uniqueString(resourceGroup().id)}'
+
+@description('Location of the data factory.')
+param location string = resourceGroup().location
+
+@description('Name of the Azure storage account that contains the input/output data.')
+param storageAccountName string = 'storage${uniqueString(resourceGroup().id)}'
+
+@description('Name of the blob container in the Azure Storage account.')
+param blobContainerName string = 'blob${uniqueString(resourceGroup().id)}'
+
+@description('Name of the managed identity created to access the Azure Storage account.')
+param managedIdentityName string = 'blob${uniqueString(resourceGroup().id)}'
+
+var dataFactoryLinkedServiceName = 'ArmtemplateStorageLinkedService'
+var dataFactoryDataSetInName = 'ArmtemplateTestDatasetIn'
+var dataFactoryDataSetOutName = 'ArmtemplateTestDatasetOut'
+var pipelineName = 'ArmtemplateSampleCopyPipeline'
+var bootstrapRoleAssignmentId_var = guid('${resourceGroup().id}contributor')
```

Comment:
rename the var to ```bootstrapRoleAssignmentName``` and use ```guid(resourceGroup().id, managedIdentity.id, roleDefinitionId)``` for the value

You also need to add a variable for:

```bicep
roleDefinitionId = subscriptionResourceId('Microsoft.Authorization/roleDefinitions', 'b24988ac-6180-42a0-ab88-20f7382dd24c')
```